s/soap-legacy/soap-legacy_ubi8.4.sh

```diff
@@ -0,0 +1,50 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : soap-legacy
+# Version       : 1.0.0-SNAPSHOT
+# Source repo   : https://github.com/reficio/soap-ws
+# Tested on     : ubi: 8.4
+# Script License: Apache License 2.0
+# Maintainer's  : Sapna Shukla<Sapna.Shukla@ibm.com>
+#
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+#Variables.
+BASE_PACKAGE_NAME=soap-ws
+PACKAGE_NAME=soap-legacy
+PACKAGE_URL=https://github.com/reficio/soap-ws
+PACKAGE_VERSION=master
+
+# Installation of required sotwares. 
+yum install git maven -y 
+
+# Cloning the repository from remote to local. 
+git clone $PACKAGE_URL
+cd $BASE_PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+# Build and test.
+if ! mvn -T 1C clean install -Dlicense.skip=true -Dadditionalparam=-Xdoclint:none -DskipTests; then
+	echo "------------------$BASE_PACKAGE_NAME:INSTALL_FAILED-------------------------------------"
+	exit 0
```

Comment:
Return value for failure should not be 0.

Reply:
Changed this to exit 1